api/routes/api.js

```diff
@@ -61,14 +54,12 @@ router.get('/regVoter/sendOTP', fetchVoter, async (req, res, next) => {
   } else {
     res.status(200).json({
       msg: 'Wrong phone number :(',
-      phonenumber: req.phone,
```

Comment:
we might use this data to show appropriate message in frontend

Reply:
We were not showing any message as of now. If we will show some message in future then we may add it.